src/meta/processors/job/BalanceJobExecutor.h

```diff
@@ -147,6 +147,28 @@ class BalanceJobExecutor : public MetaJobExecutor {
     return Status::OK();
   }
 
+  /**
+   * @brief Update last time in meta service
+   *
+   * @return
+   */
+  nebula::cpp2::ErrorCode updateLastTime() {
+    std::vector<kvstore::KV> data;
```

Comment:
Add lock here~

Reply:
the lock have been support before it be called